utils/interfaces/schemas/library/debugger/v1/diagnostics-request.json

```diff
@@ -1,4 +1,60 @@
 {
-    "$id": "/library/debugger/v1/diagnostics-request.json"
+  "$id": "/library/debugger/v1/diagnostics-request.json",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "properties": {
+      "headers": {
+        "type": "object",
+        "patternProperties": {
+          ".*": { "type": "string" }
+        }
+      },
+      "content": {
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "ddsource": { "type": "string" },
+            "debugger": { 
+              "type": "object",
+              "properties": {
+                "diagnostics" : {
+                  "type": "object",
+                  "properties": {
+                    "probeId": { "type": "string" },
+                    "probeVersion": { "type": "number" },
+                    "runtimeId": { "type": "string" },
+                    "status": { "enum": ["RECEIVED", "INSTALLED", "EMITTING", "ERROR"] }
```

Comment:
There is also `BLOCKED` (rarely used, but still)

Reply:
then I need to add it, otherwise it will break other CI when they will use it :) 

[c7d60dd](https://github.com/DataDog/system-tests/pull/2389/commits/c7d60dd8c55ba92b5e89c53e33ae31f39f7370bc)